app/views/claims/gender.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
+    <%= form_for current_claim, url: claim_path do |form| %>
+      <%= form_group_tag current_claim do %>
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+            <h1 class="govuk-fieldset__heading"><%= t("tslr.questions.gender") %></h1>
+          </legend>
+
+          <span id="gender-hint" class="govuk-hint">
+            Your school will need to send HMRC a "Male" or "Female" gender to pay you. We ask so we can provide this information to HMRC who use it along with other data to find you in their database, allowing us to make a payment to you.
+          </span>
+
+          <div class="govuk-radios">
+            <div class="govuk-radios__item">
+              <%= form.radio_button(:gender, :female, class: "govuk-radios__input") %>
+              <%= form.label :"gender_female", t("tslr.answers.gender.female"), class: "govuk-label govuk-radios__label" %>
+            </div>
+            <div class="govuk-radios__item">
+              <%= form.radio_button(:gender, :male, class: "govuk-radios__input") %>
+              <%= form.label :"gender_male", t("tslr.answers.gender.male"), class: "govuk-label govuk-radios__label" %>
+            </div>
+            <div class="govuk-radios__divider">or</div>
+            <div class="govuk-radios__item">
+              <%= form.radio_button(:gender, :dont_know, class: "govuk-radios__input") %>
+              <%= form.label :"gender_dont_know", t("tslr.answers.gender.dont_know"), class: "govuk-label govuk-radios__label" %>
+            </div>
+          </div>
+        </fieldset>
+      <% end %>
+      <%= form.submit "Continue", class: "govuk-button" %>
+
+      <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">
+            If you don't know
+          </span>
+        </summary>
+        <div class="govuk-details__text">
+          If you don't know which gender is held on record we will contact HMRC or the Teachers Pension Scheme. We don't need to record this information but it is required to make tax contributions on your behalf to HMRC.
```

Comment:
Suggest we use "what" instead of "which" when talking about gender. Feels like that calls out the fact that we understand gender is more than a binary choice.